client/my-sites/stats/site.jsx

```diff
@@ -204,44 +218,86 @@ class StatsSite extends Component {
 						}
 					) }
 				/>
+
 				<StatsNavigation
 					selectedItem="traffic"
 					interval={ period }
 					siteId={ siteId }
 					slug={ slug }
 				/>
 
-				{ isSitePrivate ? this.renderPrivateSiteBanner( siteId, slug ) : null }
-				{ ! isSitePrivate && <StatsNoContentBanner siteId={ siteId } siteSlug={ slug } /> }
-
-				<div id="my-stats-content">
-					<ChartTabs
-						activeTab={ getActiveTab( this.props.chartTab ) }
-						activeLegend={ this.state.activeLegend }
-						availableLegend={ this.getAvailableLegend() }
-						onChangeLegend={ this.onChangeLegend }
-						barClick={ this.barClick }
-						switchTab={ this.switchChart }
-						charts={ CHARTS }
-						queryDate={ queryDate }
-						period={ this.props.period }
-						chartTab={ this.props.chartTab }
-					/>
-					<StickyPanel className="stats__sticky-navigation">
-						<StatsPeriodNavigation
-							date={ date }
-							period={ period }
-							url={ `/stats/${ period }/${ slug }` }
-						>
-							<DatePicker
+				{ isNewFeatured ? (
```

Comment:
Perhaps we should wrap this entire block inside div#my-stats-content?

```suggestion
				<div id="my-stats-content">
					{ isNewFeatured ? (
```

Reply:
I think so since I could not find the usage related to the id `#my-stats-content`, the moving may be less risky.

Reply:
Resolved in [d562603](https://github.com/Automattic/wp-calypso/pull/69529/commits/d5626033959d08feb66407e5fb8091d5c0916bee).